.docker/browser/translations/messages+intl-icu.en.yaml

```diff
@@ -1,2 +1,3 @@
 libero.page:
+    info_bar: This is a demo.
```

Comment:
The `demo` environment will need a different message, not sure how that would be separated.

Reply:
Multiple messages here and different environment variables?

Reply:
Maybe https://github.com/libero/sample-configuration/blob/afe8b80653102299c32eba1715970768dfdb14b2/docker-compose.yml#L64 should be a default?

Reply:
https://github.com/libero/environments/pull/42 added support for different environment variables in different environments, may help. Though for actual text and complex configurations we should probably have separate configuration files instead?

Reply:
If the value is just `libero.page.info_bar` good to go as an env variable, it won't scale to too many variables like this however.